eventdef/eventdef.go

```diff
@@ -57,12 +86,77 @@ func Save(e *schema.ProbeEvent) error {
 	if err := e.Validate(); err != nil {
 		return err
 	}
+	
+	t := time.Now()
+	y, m, d := t.Date()
+	idxName := fmt.Sprintf("events-%d-%02d-%02d", y, m, d)
+	if err := checkIdx(idxName, e); err != nil {
```

Comment:
You dont need to do this.. Use a index template in Elasticsearch and the index will be automatically created when first written to.
https://www.elastic.co/guide/en/elasticsearch/reference/1.6/indices-templates.html